core/build.gradle

```
@@ -4,11 +4,18 @@ plugins {
     id 'org.jetbrains.kotlin.jvm'
 }
 
+ext {
+    PUBLISH_NAME = 'Amplitude Kotlin Core'
+    PUBLISH_DESCRIPTION = 'Amplitude Kotlin core library'
```

```suggestion
    PUBLISH_DESCRIPTION = 'Amplitude Kotlin Core Library'
```

for uniformity with the other descriptions